exporter/logzioexporter/exporter_test.go

```diff
@@ -92,6 +91,21 @@ func fillLogNoTimestamp(log plog.LogRecord) {
 	log.Body().SetStr("something happened")
 }
 
+func fillLogScopeName(log plog.LogRecord) {
+	log.SetTimestamp(TestLogTimestamp)
+	log.SetDroppedAttributesCount(2)
+	log.SetSeverityNumber(plog.SeverityNumberInfo)
+	log.SetSeverityText("Info")
+	log.SetSpanID([8]byte{0x01, 0x02, 0x04, 0x08})
+	log.SetTraceID([16]byte{0x08, 0x04, 0x02, 0x01})
+	attrs := log.Attributes()
+	attrs.PutStr("app", "log4j2")
+	attrs.PutDouble("instance_num", 1)
+	attrs.PutStr("scopeName", "test.class")
```

Comment:
That’s not where scope name usually goes. Look at scope records, the parent of the log record.

Reply:
@atoulme @codeboten Care to provide a more detailed explanation? :)

Reply:
There is a type named `ScopeLogRecords` which contains the "scope" attributes and should exist in that type.

You can see where you have used it in the `exporter` file :)